main.go

```diff
@@ -84,6 +89,28 @@ func run() error {
 
 	ps := store.NewPebbleStore(db, nil)
 
+	if cfg.EmptyTicks.CalculateAll == true {
+
```

Comment:
using context.Background is bad, the task should have a timeout so you need to use a proper context